app/cdap/components/common/DrawerWidget/index.tsx

```diff
@@ -110,15 +112,19 @@ export default function({
       <DrawerHeader>
         <HeaderActions>
           {showBackIcon && (
-            <CustomizedIconButton aria-label="back-icon" data-testid="back-icon">
+            <CustomizedIconButton
+              aria-label="back-icon"
+              data-testid="back-icon"
+              onClick={onBackIconClick}
+            >
               <BackIcon />
             </CustomizedIconButton>
           )}
           <LabelContainer>
             <Label data-testid="widget-heading-text" component="span">
               {headingText}
             </Label>
-            <img src="/cdap_assets/img/underline" />
+            {/* <img src="/cdap_assets/img/underline" /> */}
```

Comment:
?

Reply:
In latest UX spec version3, header separator is not shown. But Divami's team is still working on UX version 2 for few wrangler UIs which show separator.  Changed as parameterised  for now as we need to support both it seems .